ui/src/app/views/dataset-page/component.tsx

```diff
@@ -89,15 +90,33 @@ export const ValueListItem: React.FunctionComponent <
     </div>;
   };
 
-export const DataSetPage = withCurrentWorkspace()(class extends React.Component<
-  {workspace: WorkspaceData},
-  {creatingConceptSet: boolean, conceptDomainList: DomainInfo[],
-    conceptSetList: ConceptSet[], cohortList: Cohort[], loadingResources: boolean,
-    confirmDeleting: boolean, editing: boolean, resource: RecentResource,
-    rType: ResourceType, selectedConceptSetIds: number[], selectedCohortIds: number[],
-    valueSets: ValueSet[], selectedValues: DomainValuePair[], openSaveModal: boolean,
-    queries: Array<DataSetQuery>, includesAllParticipants: boolean
-  }> {
+interface Props {
+  workspace: WorkspaceData;
+}
+
+interface State {
+  confirmDeleting: boolean;
+  cohortList: Cohort[];
+  conceptDomainList: DomainInfo[];
+  conceptSetList: ConceptSet[];
+  creatingConceptSet: boolean;
+  dataSets: Array<DataSetPreviewList>;
```

Comment:
Nit: Can we call this `dataSetTablePreviews` or something? I just picture this as multiple data sets as currently written.